core/src/test/java/org/elasticsearch/search/builder/SearchSourceBuilderTests.java

```diff
@@ -663,4 +663,25 @@ public void testEmptyQuery() throws IOException {
         String query = "{ \"query\": {} }";
         assertParseSearchSource(builder, new BytesArray(query), ParseFieldMatcher.EMPTY);
     }
+
+    public void testSearchRequestBuilderSerializationWithIndexBoost() throws Exception {
```

Comment:
I think we could have add some randomization for index boost to createSearchSourceBuilder rather than adding a specific test method for it?